cdap-ui/app/cdap/components/FastAction/SetPreferenceAction/SetPreferenceAction.scss

```diff
@@ -240,7 +283,7 @@ $link-color: #0275d8;
 
               tbody {
                 display: block;
-                overflow: scroll;
+                overflow: auto;
```

Comment:
Not sure what you mean by container overflow, but here I'm limiting the number of content `tr`s that show on the screen, while keeping the headers in view, and `tbody` is the container of the content `tr`s. 

It doesn't work as expected if I set the `overflow: auto` anywhere else.